modules/recommended-configurable-storage-technology.adoc

```diff
@@ -121,8 +121,19 @@ It is not recommended to use file storage for a hosted metrics cluster deploymen
 
 In an {product-title} hosted logging cluster deployment:
 
-* The preferred storage technology is block storage.
-* Object storage is not configurable.
+* Loki Operator:
+** The preferred storage technology is S3 compatible Object storage.
+** Block storage is not configurable.
+
+* Elasticsearch Operator:
+** The preferred storage technology is block storage.
+** Object storage is not supported.
+
+[NOTE]
+====
+As of logging version 5.4.3 the OpenShift Elasticsearch Operator is deprecated and is planned to be removed in a future release. Red Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed. As an alternative to using the OpenShift Elasticsearch Operator to manage the default log storage, you can use the Loki Operator.
```

Comment:
In most cases I would say this needs revised because:

1. There is an attribute for this product (that is, Elasticsearch Operator), and it's {es-os}. But we don't ever use it.
2. We're inconsistent with our usage of "OpenShift Elasticsearch Operator", "Elasticsearch Operator", and "Red Hat OpenShift Elasticsearch Operator" and some consistency would be nice.

That said,  I'm not sure if any of this matters because it's being deprecated and removed anyways.

Reply:
My mistake, should have specified. This text comes from [this snippet](https://github.com/openshift/openshift-docs/blob/main/snippets/logging-elastic-dep-snip.adoc) but I didn't want to `include` a snippet in docs I don't own because sometimes they have unpredictable outcomes. :)